app/javascript/src/Dashboard/ajax-widget.jsx

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import { widget } from 'Dashboard'
+
+export function loadAudienceWidget (widgetPath: string) {
+  widget(widgetPath)
```

Comment:
Maybe you could

`import { widget as makeWidgetRequest } from 'Dashboard'`

Or something similar

Reply:
I like that 👍